datamaker/core/storage_providers/__init__.py

```diff
@@ -0,0 +1,19 @@
+from abc import abstractmethod, ABC
+from typing import Optional
+
+
+class BaseStorageProvider(ABC):
+    @property
+    @abstractmethod
+    def label(self):
+        pass
+
+    def __init__(self, configuration: Optional[dict] = None):
+        if configuration is None:
+            configuration = {}
+
+        self.configuration = configuration
+
+    @abstractmethod
+    def get_pathlib(self):
+        raise NotImplementedError
```

Comment:
[P3]
`@abstractmethod`를 쓴 이상 `raise NotImplementedError`는 불필요한 코드입니다.
```suggestion
        pass
```

Reply:
이거 해결이 안되셨습니다

Reply:
아 아래  부분까지 체크되면 push하려고 했던 부분이긴한데 지금 Push하겠습니다!!